src/transformers/pipelines/__init__.py

```diff
@@ -199,26 +201,22 @@
         "pt": (AutoModelForVisualQuestionAnswering,) if is_torch_available() else (),
         "tf": (),
         "default": {
-            "model": {
-                "pt": "dandelin/vilt-b32-finetuned-vqa",
-                "tokenizer": "dandelin/vilt-b32-finetuned-vqa",
-                "feature_extractor": "dandelin/vilt-b32-finetuned-vqa",
```

Comment:
deleted tokenizer and feature extractor because of https://github.com/huggingface/transformers/pull/17667/files#r910987282